### ISSUE TITLE
Pipeline push to DynamoDB for Charting

### ISSUE THREAD

**Issue by sashahart** [TNRIS/flood#72]
This depends on #71 so we can figure out what we need to get from where.

**Comment by sashahart** [comment]
I'd like to clarify that this issue was meant to discuss data supply for charting purposes, as distinct from CartoDB which will directly supply map layers. 

Just to be explicit, the composition of the system that is pushing data to CartoDB is I think a tactical issue that isn't in scope for this particular issue.

**Comment by sashahart** [comment]
Closing in favor of a new issue since "data service" is not a real thing or requirement
